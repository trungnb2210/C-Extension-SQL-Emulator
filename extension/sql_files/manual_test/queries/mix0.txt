SELECT item, MAX(price), AVG(price)
FROM things JOIN prices ON object == item
GROUP BY item
HAVING MAX(price) > 150
ORDER BY item, MAX(price)
