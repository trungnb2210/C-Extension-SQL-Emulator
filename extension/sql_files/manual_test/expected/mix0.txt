QUERY RESULT:
  item    MAX(price)   AVG(price) 
-----------------------------------
| Box   | 250        | 175        | 
| Table | 300        | 300        | 
-----------------------------------
